optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/common/TimeSlotUtils.java

```diff
@@ -7,6 +7,10 @@
 
 public class TimeSlotUtils {
 
+    public static LocalDate toLocalDate(OffsetDateTime time) {
+        return LocalDate.of(time.getYear(), time.getMonthValue(), time.getDayOfMonth());
```

Comment:
(hard) Use OffsetDateTime.toLocalDate()

Reply:
I would. but our emulated libary does not have ` OffsetDateTime.toLocalDate()` (causes a GWT compile error)